### PR TITLE
New onboarding flow.

### PR DESCRIPTION
**Pull request by niwinz**

### CODE REVIEW

File: frontend/src/app/main/ui.cljs [hunk 1]
```diff
@@ -72,7 +72,15 @@
         :dashboard-team-settings)
        [:*
         #_[:div.modal-wrapper
-           [:& app.main.ui.onboarding/release-notes-modal {:version "1.8"}]]
+           ;; modal de la derecha
```
Comment (hirunatan) [inline frontend/src/app/main/ui.cljs]: In spanish? :smile:

File: frontend/src/app/main/ui/onboarding.cljs [hunk 2]
```diff
@@ -187,56 +192,232 @@
...
+                     {:type :import
+                      :project-id project-id
+                      :files [file]
+                      :on-finish-import on-finish-import})))
+        on-click
+        (fn []
+          (reset! downloading? true)
+          (->> (http/send! {:method :get :uri link :response-type :blob :mode :no-cors})
+               (rx/subs (fn [{:keys [body] :as response}]
+                          (js/console.log (pr-str response))
```
Comment (hirunatan) [inline frontend/src/app/main/ui/onboarding.cljs]: This trace is ok?

File: frontend/translations/en.po [hunk 3]
```diff
@@ -3187,4 +3199,5 @@ msgid "workspace.updates.update"
 msgstr "Update"
 
 msgid "workspace.viewport.click-to-close-path"
-msgstr "Click to close the path"
\ No newline at end of file
+msgstr "Click to close the path"
```
Comment (hirunatan) [inline frontend/translations/en.po:3202]: You should also create the texts in spanish at least.

File: frontend/src/app/main/ui.cljs (hunk 1 above)
Comment (niwinz) [reply frontend/src/app/main/ui.cljs]: yeah, that was a quick and dirty comments for the @elhombretecla  to know which component to touch :P

File: frontend/translations/en.po (hunk 3 above)
Comment (niwinz) [reply frontend/translations/en.po:3202]: The spanish texts, can be handled and is usually handled directly in weblate. So, unless we have spanish texts on designs, I think is better leave it to be translated later...

File: frontend/src/app/main/ui/onboarding.cljs (hunk 2 above)
Comment (niwinz) [reply frontend/src/app/main/ui/onboarding.cljs]: :fire:

**Pull request merged**